DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/MVP.Script.txt

```diff
@@ -78,45 +69,49 @@ Text GetManialink() {
 	Integer[CSmScore] GetMatchPoints() {
 		declare Integer[CSmScore] Output;
 		foreach(Score in Scores) {
-			declare netread Integer Net_FlagRush_MatchPoints for Score;
-			//if (Net_FlagRush_MatchPoints > 0) 
-			Output[Score] = Net_FlagRush_MatchPoints;			
-			
+			declare netread Integer Net_FlagRush_Points_Match for Score;
+			Output[Score] = Net_FlagRush_Points_Match;			
 		}
 		return Output.sortreverse();
 	}
 	
 	/**
 	* Get Best player of FlagsScored
 	*/
-	CSmScore GetFlagsScored() {
-		declare Integer[CSmScore] Output;
-		foreach(Score in Scores) {
+	CSmScore GetScoreMostFlagsScored() {
+		declare CSmScore BestScore;
+		declare Integer MaxFlagsScored = 0;
+		foreach (Score in Scores) {
 			declare netread Integer Net_FlagRush_FlagsScored_Match for Score;
-			Output[Score] = Net_FlagRush_FlagsScored_Match;			
+			if (Net_FlagRush_FlagsScored_Match > MaxFlagsScored) {
+				BestScore <=> Score;
+				MaxFlagsScored = Net_FlagRush_FlagsScored_Match;
+			}
 		}
-		Output = Output.sortreverse();
-		return GetScore(Output, 0);
```

Comment:
Function `GetScore()` (line 55-64) is not used anymore. Please remove it. (I couldn't put the comment on an unchanged line, thanks GitHub.)